docs/rules/ExplicitResultTypes.md

```diff
@@ -34,13 +34,6 @@ This rule has several known limitations, which are most likely fixable with some
 effort. At the time of this writing, there are no short-term plans to address
 these issues however.
 
-### Limited Scala support
-
-Due to implementation details, this rule works only with @SCALA212@. Scala 2.11
-and 2.13 are not supported for this rule. See
-[#988](https://github.com/scalacenter/scalafix/issues/998) if you want to
-contribute support for other Scala versions.
-
```

Comment:
Should we keep this until the rule itself is fully functionnal with 2.13? Since my intention was not to close https://github.com/scalacenter/scalafix/issues/998 with this, I guess so...

Reply:
I guess we can remove it now that https://github.com/scalacenter/scalafix/pull/1167 is going to close #998

Reply:
Agree. 
I